scheduler/clippy.toml

```diff
@@ -0,0 +1,2 @@
+allow-unwrap-in-tests = true
+allow-expect-in-tests = true
```

Comment:
We will allow `unwrap` and `expect` in tests, as we don't need to be "as safe" in them (just like we sometimes use `any` in TS tests)